Example/Example/ContentView.swift

```diff
@@ -61,15 +60,32 @@ struct ContentView: View {
                 Text("Hide/Show TabBar")
             }
             .tabItem(for: Item.first)
-            
-            Text("Second")
+
+            TextWrapper()
                 .tabItem(for: Item.second)
-            
-            Text("Third")
+
+            TextWrapper()
                 .tabItem(for: Item.third)
         }
         .tabBar(style: CustomTabBarStyle())
         .tabItem(style: CustomTabItemStyle())
+        .onChange(of: selection) { newValue in
+            print("selection changed:", newValue)
+        }
+    }
+}
+
+struct TextWrapper: View {
+    @State var string: String = UUID().uuidString
+
+    var body: some View {
+        Text(string)
+            .onTapGesture {
+                string = UUID().uuidString
+            }
+            .onAppear(perform: {
+                print("onAppear:", string)
+            })
     }
 }
```

Comment:
Could you please clarify the purpose of this wrapper? I didn't quite understand it.

Reply:
Just for the purpose of testing recreation and lazy loading,  no other meaning.
If you don't like it, I can minimize the PR.

Reply:
I think it will be better revert these changes, as Example project most likely will be updated later.